apps/docs/components/collapsible/Collapsible.tsx

```diff
@@ -3,7 +3,7 @@
 import { CollapseIcon, Icon } from "@/components/icon";
 import clsx from "clsx";
 import type { ReactNode } from "react";
-import { Button, composeRenderProps, UNSTABLE_Disclosure as Disclosure, UNSTABLE_DisclosurePanel as DisclosurePanel, type DisclosureProps } from "react-aria-components";
+import { Button, composeRenderProps, Disclosure, DisclosurePanel, type DisclosureProps } from "react-aria-components";
 
```

Comment:
<img width="912" alt="image" src="https://github.com/user-attachments/assets/c9a4c0a2-ad22-487d-9727-d9e0c36f531e">
I wonder if the content padding should be more proeminent 
<img width="910" alt="image" src="https://github.com/user-attachments/assets/3b1c0a8e-d674-4a30-babb-7f0205cc2e79">
Especially when there's an icon.

Reply:
no I don't think so... and I went be what there was in figma

Reply:
Figma had no visible examples, my bad.